inc/templates/admin-bar.php

```diff
@@ -348,3 +349,50 @@ function get_documentation_items(): array {
 		],
 	];
 }
+
+/**
+ * Get an array of Irving cache busting node args to be passed into
+ * admin_bar->add_node().
+ *
+ * @return array
+ */
+function get_cache_items(): array {
```

Comment:
Is there a way to extend this to enable "Site Wide" on pantheon (where it's supported it), and also a "This page only" version for Patheon + VIP?